{{cookiecutter.pipeline_slug}}/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: nfcore-{{ cookiecutter.pipeline_name }}
+dependencies:
+  - fastqc=0.11.5
+  - multiqc=1.5a
+channels:
```

Comment:
It feels more natural to have this above `dependencies` personally. Also, I usually add in `defaults` at the top so that it's explicit that the other channels have a higher rank (though I don't think it makes any difference).